packages/anon-aadhaar-react/src/provider/AnonAadhaarProvider.tsx

```diff
@@ -22,16 +22,24 @@ import { SerializedPCD } from '@pcd/pcd-types'
  * @returns A JSX element that wraps the provided child components with the
  * AnonAadhaarContext.Provider.
  */
-export function AnonAadhaarProvider(props: { children: ReactNode }) {
+export function AnonAadhaarProvider(props: {
+  children: ReactNode
+  _appId: string
+}) {
   // Read state from local storage on page load
-  const [pcdStr, setPcdStr] = useState<SerializedPCD<AnonAadhaarPCD> | ''>('')
-  const [pcd, setPcd] = useState<AnonAadhaarPCD | ''>('')
+  const [pcdStr, setPcdStr] = useState<SerializedPCD<AnonAadhaarPCD> | null>(
+    null,
+  )
+  const [pcd, setPcd] = useState<AnonAadhaarPCD | null>(null)
+  const [appId, setAppId] = useState<string | null>(null)
   const [state, setState] = useState<AnonAadhaarState>({
     status: 'logged-out',
   })
   useEffect(() => {
     readFromLocalStorage().then(setAndWriteState)
-  }, [])
+    setAppId(props._appId)
+    console.log(props._appId)
```

Comment:
I think we should have Loglevel for debug :D

Reply:
Good catch, I missed this one!